src/shippingservice/Cargo.toml

```diff
@@ -13,27 +13,27 @@ path = "src/main.rs"
 dockerproto = [] # compile in docker or not
 
 [dependencies]
-tonic = "0.10"
-tonic-health = "0.10.2"
+tonic = "0.11"
+tonic-health = "0.11.0"
 reqwest = { version = "0.11", default-features = false, features = ["json"] }
 prost = "0.12"
 prost-types = "0.12"
-tokio = { version = "1.34", features = ["macros", "rt-multi-thread"] }
-opentelemetry = { version = "0.21", features = ["trace"] }
-opentelemetry_sdk = {version = "0.21", features = ["rt-tokio"]}
+tokio = { version = "1.36", features = ["macros", "rt-multi-thread"] }
+opentelemetry = { version = "0.21", features = ["trace", "metrics", "logs"] }
+opentelemetry-appender-log = "0.2.0"
```

Comment:
I'd recommend to use `tracing` and `opentelemetry-appender-tracing`. 
`log` crate does not support structured logging, and we want to showcase best practices in the demo, right?